src/Components/AllReviews.jsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react";
+import { ReviewCard } from "./ReviewCard";
+
+export function AllReviews() {
+  const [reviews, setReviews] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const reqURL = "https://board-game-reviews-brfi.onrender.com/api/reviews";
+
+  useEffect(() => {
+    getAllReviews().then(() => {
+      setIsLoading(false);
+    });
+  }, []);
+
+  const getAllReviews = async () => {
+    const response = await fetch(`${reqURL}`);
+    const { reviews } = await response.json();
+    setReviews(reviews);
+  };
+
+  return isLoading ? (
+    <main>
+      <section>
+        <h2>All reviews</h2>

```

Comment:
Theres a fair bit of code duplication in this ternery here, it might be nice to use the ternery underneath the h2 instead to either render the loading message or the reviews just to prevent this.

Reply:
I did think of this, but I didn't know how to do it. I'll look into it!